gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
```

Comment:
Gradle'ın Java 11 gereksinimi olduğundan Github Action's için güncelleme yapılması gerekli. mevcut config 1.8 kullanıyor.

Reply:
Java 11 GitHub Actions entegrasyonu yapıldı